src/win32.mak

```diff
@@ -377,8 +380,11 @@ pvs:
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$C;$(TK) /Tp $(BACKSRC) --source-file $(BACKSRC)
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$(TK) /Tp $(TKSRCC) --source-file $(TKSRCC)
 
-checkwhitespace:
-	$(HOST_DC) -run checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+checkwhitespace: $(TOOLS_DIR)
+	$(HOST_DC) -run $(TOOLS_DIR)\checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+
+$(TOOLS_DIR):
+	git clone --depth=1 $(GIT_HOME)/tools $@
```

Comment:
I really hate all this redundancy going on in the `win{32,64}` Makefile.
Isn't it feasible to 

A) Unify the `win32.mak` and `win64.mak` files into one? 
(there's `if` in make and it's possible to detect 64-bit as well, especially for Phobos this would help a lot)

B) Unify _all_ Makefile into one source of truth?

We could define the path `$S` symbol and use it commonly like e.g. `my${S}path${S}`